tests/PKSim.Tests/IntegrationTests/SimulationWithParticlesFormulationSpecs.cs

```diff
@@ -448,6 +484,160 @@ private void checkValuesMonotonous(float[] values, bool checkDecreasing)
       }
    }
 
+   public class when_comparing_with_prototype_simulation_in_mobi : concern_for_SimulationWithParticlesFormulation
```

Comment:
Here some relevant simulated values are compared between a prototypical simulation manually created in MoBi and a new simulation created in PK-Sim